pkg/reco/recommenders.go

```diff
@@ -0,0 +1,173 @@
+package reco
```

Comment:
Should we merge reco.go and recommenders.go into a single file?

Reply:
Yes. recommenders.go was initially named as reco.go . I renamed it just to simplify my merging with main to avoid conflicts. Can be merged.